vue-components/stories/Button.stories.ts

```diff
@@ -21,3 +21,17 @@ export function normal(): Component {
 		`,
 	};
 }
+
+export function quiet(): Component {
```

Comment:
Once #329 and #324 are merged, we'll create separate pages for iconOnly buttons per variant ([see structure](https://docs.google.com/document/d/1loENZ13z5s_Nes4exm5Z9d5S3PnR7F_0brv5azQ5XB4/edit)). Happy to help with that 🙋‍♀️